trust-x-htb.js

```diff
@@ -460,6 +480,14 @@ function TrustXHtb(configs) {
             parseResponse: __parseResponse,
             generateRequestObj: __generateRequestObj
         });
+
+        /* If wrapper is already active, we might be instantiated late so need to add our callback
+         since the shell potentially missed its chance */
+        if (window[SpaceCamp.NAMESPACE]) {
```

Comment:
Without this logic we can't test our adapter in adapter-debugger, because the headertag has no property of our adapter - headertag.TrustXHtb is undefined and our request callback is not called. We are not aware of other possible solutions to this problem, so please advise if there are any.